Libraries/LibWeb/Parser/CSSParser.cpp

```diff
@@ -136,20 +136,23 @@ static Optional<float> parse_number(const StringView& view)
 
 NonnullRefPtr<StyleValue> parse_css_value(const StringView& string)
 {
-    auto number = parse_number(string);
+    auto string_lowercase = string.to_string().to_lowercase();
```

Comment:
Same here, we can use `equals_ignoring_case` and avoid creating a temporary string.